readthedocs/proxito/tests/responses/v0.json

```diff
@@ -0,0 +1,120 @@
+                }
+                ]
+            },
+            "version": {
+                "active": true,
+                "hidden": false,
+                "built": true,
+                "downloads": {},
+                "id": 1,
+                "identifier": "a1b2c3",
+                "ref": null,
+                "slug": "latest",
+                "type": "tag",
+                "verbose_name": "latest"
+            },
+            "build": {
+                "commit": "a1b2c3",
+                "created": "2019-04-29T10:00:00Z",
+                "duration": 60,
+                "error": "",
+                "finished": "2019-04-29T10:01:00Z",
+                "id": 1,
+                "project": "project",
+                "state": {
+                    "code": "finished",
+                    "name": "Finished"
+                },
+                "success": true,
+                "version": "latest"
+            },
+            "domains": {
+                "dashboard": "readthedocs.org"
+            },
+            "readthedocs": {
+                "analytics": {
+                    "code": null
+                }
+            },
+            "features": {
+                "analytics": {
+                    "code": null
+                },
+                "external_version_warning": {
+                    "enabled": true,
+                    "query_selector": "[role=main]"
+                },
+                "non_latest_version_warning": {
+                    "enabled": true,
+                    "query_selector": "[role=main]",
+                    "versions": [
+                        "latest"
+                    ]
+                },
+                "doc_diff": {
+                    "enabled": true,
+                    "base_url": "https://project.dev.readthedocs.io/en/latest/index.html",
+                    "root_selector": "[role=main]",
+                    "inject_styles": true,
+                    "base_host": "",
+                    "base_page": ""
+                },
+                "flyout": {
+                    "translations": [],
+                    "versions": [
+                        {"slug": "latest", "url": "/en/latest/"}
+                    ],
+                    "downloads": [],
+                    "vcs": {
+                        "url": "https://github.com",
+                        "username": "readthedocs",
+                        "repository": "test-builds",
+                        "branch": "a1b2c3",
+                        "filepath": "/docs/index.rst"
+                    }
+                },
+                "search": {
+                    "api_endpoint": "/_/api/v3/search/",
+                    "default_filter": "subprojects:project/latest",
+                    "filters": [
+                        ["Search only in this project", "project:project/latest"],
+                        ["Search subprojects", "subprojects:project/latest"]
```

Comment:
I like this pattern. It also helps push the translations into the application which I like. Translations will be A Thing though, so we can punt on that conversation either way.

Reply:
Well, this is just a hard coded/default example.

In theory, what I understood from the work that @stsewd did, is that _the author of the docs_ is whom will decide the names and the values of the filters. So, we need to keep talking how that data will be passed (see https://github.com/readthedocs/readthedocs-client/issues/13)